frontend/src/app/components/enterprise/enterprise-modal/enterprise-trial-form/ee-trial-form.component.ts

```diff
@@ -31,9 +31,8 @@ import {FormBuilder, Validators} from "@angular/forms";
 import {I18nService} from "app/modules/common/i18n/i18n.service";
 import {EnterpriseTrialData, EnterpriseTrialService} from "core-components/enterprise/enterprise-trial.service";
 import {CurrentUserService} from "core-components/user/current-user.service";
+import {I18nHelpers} from "core-app/helpers/i18n/localized-link";
 
-const termsOfServiceURL = 'https://www.openproject.com/terms-of-service/';
-const legalNoticeURL = 'https://www.openproject.com/legal-notice/';
 const newsletterURL = 'https://www.openproject.com/newsletter/';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/b9b593f63f50130652ab0ee1ff788f085be19ea8/frontend/src/app/components/enterprise/enterprise-modal/enterprise-trial-form/ee-trial-form.component.ts#L37"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>